### PR TITLE
openflow_acl_test: replace lynx with curl for checking website status 

### PR DESCRIPTION
**Pull request by pezhang**
id: 1467779
Signed-off-by: Pei Zhang <pezhang@redhat.com>

### CODE REVIEW

**Comment by xiywang** [comment]
why replacing lynx with wget?
lynx is a web browser which is used to open http://ACCESS_TARGET/autotest-test.html.
If you use wget and set output to /dev/null, eg: # wget http://ab.com/a --output-document=/dev/null
abviously, this file is not exist but the prompt of this command is: 2017-08-14 08:23:35 (204 MB/s) - ‘/dev/null’ saved [734/734].
And when # echo $?, the result is 0, which means succeed.
Then how can you tell whether the httpd server works well?

**Comment by pezhang** [comment]
@xiywang   Thanks for review. In this example, http://ab.com  is an valid address, that's why echo $? returns 0. Please try http://ab.comx/a, then you will get non-zero value.

In this case, if the server down, then wget http://ACCESS_TARGET/autotest-test.html will fail, echo $0 will return non-zero value.

As lynx is not easy to install in our current framework, so this method could be considered to be an alternative way.

Please check again. Thanks.

**Comment by xiywang** [comment]
Thanks for explanation.
 Acked-by: Xiyue Wang xiywang@redhat.com

File: qemu/tests/cfg/openflow_acl_test.cfg [hunk 1]
```diff
@@ -14,7 +14,7 @@
             target_port = 80
             prepare_cmd_linux = echo "autotest page" > /var/www/html/autotest-test.html
             cleanup_cmd_linux = rm -rf /var/www/html/autotest-test.html
-            access_cmd_linux = lynx --dump http://ACCESS_TARGET/autotest-test.html
+            access_cmd_linux = wget http://ACCESS_TARGET/autotest-test.html --output-document=/dev/null
```
Comment (luckyh) [inline qemu/tests/cfg/openflow_acl_test.cfg]: `curl` is a better alternative for this case.

Comment (pezhang) [inline qemu/tests/cfg/openflow_acl_test.cfg]: @luckyh Thanks for review. I agree, already updated to curl. Please help to review again. Thanks.

**Comment by pezhang** [comment]
PASS 1-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.openflow_acl_test.access_deny_special_target.website

File: qemu/tests/cfg/openflow_acl_test.cfg [hunk 2]
```diff
@@ -76,7 +76,7 @@
             ftp:
                 setup_targets = localhost
             website:
-                access_cmd_linux = lynx --dump www.redhat.com
+                access_cmd_linux = curl www.redhat.com
```
Comment (luckyh) [inline qemu/tests/cfg/openflow_acl_test.cfg]: Suggest adding option `-L` to enable relocation, or you will get
```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://www.redhat.com/">here</a>.</p>
<hr>
<address>Apache Server at www.redhat.com Port 80</address>
</body></html>
```

Comment (pezhang) [inline qemu/tests/cfg/openflow_acl_test.cfg]: @luckyh Thanks, will update.

**Comment by pezhang** [comment]
Re-test again:

smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.openflow_acl_test.access_deny_special_target.website: PASS (981.27 s)

File: qemu/tests/cfg/openflow_acl_test.cfg (hunk 2 above)
Comment (xiangchunfu) [inline qemu/tests/cfg/openflow_acl_test.cfg]: when sebserver doesn't work, curl command will be in a wait state. so I suggest add this option --connect-timeout.  eg.  curl -l http://10.10.10.10 --connect-timeout 10.

Comment (pezhang) [inline qemu/tests/cfg/openflow_acl_test.cfg]: @xiangchunfu Thanks for review. Make sense. Already update the code. Please help to review again. Thanks.

**Comment by pezhang** [comment]
Re-test again:
smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.openflow_acl_test.access_deny_special_target.website: PASS (92.87 s)

**Comment by xiangchunfu** [comment]
Acked-by:Xiangchun Fu xfu@redhat.com

**Comment by luckyh** [comment]
The code looks good to me. Then, please rewrite the commit message by following:
http://avocado-framework.readthedocs.io/en/53.0/ContributionGuide.html#git-workflow

Thanks

**Comment by pezhang** [comment]
@luckyh Thanks. Have updated. Please check.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**